src/compat/libc/string/ffs.c

```diff
@@ -0,0 +1,16 @@
+/**
+ * @file
+ * @brief Implementation of ffs() function.
+ *
+ * @date 20.03.15
+ * @author Byeonggon Lee
+ */
+
+#include <strings.h>
+
+int ffs(int in) {
+	for (int i = 0; i < sizeof(int) * 8; ++i)
+		if (in & (1 << i))
+			return i + 1;
```

Comment:
Can you please use braces? https://github.com/embox/embox/wiki/Code-Style#tabs-and-brackets